tokens.xml

```diff
@@ -8460,7 +8459,7 @@ Sacrifice this creature: This creature deals 1 damage to target creature or play
             <reverse-related>Liliana, Death's Majesty</reverse-related>
             <reverse-related>Liliana, Dreadhorde General</reverse-related>
             <reverse-related>Liliana, Heretical Healer</reverse-related>
-            <reverse-related count="x">Liliana, the Last Hope</reverse-related>
+            <reverse-related count="x">Liliana, the Last Hope (Emblem)</reverse-related>
```

Comment:
The minimum number of zombies is 2, so what about `"x=2"` here?

Reply:
Sure, that works fine.

The x count and it's "correct usage" in various situations is still a bit obscure.
No consensus in the file or the people working with it. That makes it a bit tricky to find the "right" solution as there are often several ways to utilize it.